sky/services/ns_net/url_loader_impl.mm

```diff
@@ -32,6 +32,17 @@ - (void)connection:(NSURLConnection*)connection
   _response->status_code = response.statusCode;
   _response->url =
       mojo::String(self.originalRequest.URL.absoluteString.UTF8String);
+  NSUInteger headerCount = response.allHeaderFields.count;
+  if (headerCount > 0) {
+    _response->headers = mojo::Array<mojo::HttpHeaderPtr>::New(0);
```

Comment:
mojo::Array<T>::New does a resize instead of a reserve. So I don't think I can use the count here and push_back.